packages/core/src/awsService/cloudWatchLogs/registry/liveTailSession.ts

```diff
@@ -102,4 +134,15 @@ export class LiveTailSession {
             logEventFilterPattern: this.logEventFilterPattern ? this.logEventFilterPattern : undefined,
         })
     }
+
+    public showStatusBarItem(shouldShow: boolean) {
+        shouldShow ? this.statusBarItem.show() : this.statusBarItem.hide()
+    }
+
+    public updateStatusBarItemText() {
+        const elapsedTime = this.getLiveTailSessionDuration()
+        const timeString = convertToTimeString(elapsedTime)
+        const sampledString = this._isSampled ? 'Yes' : 'No'
+        this.statusBarItem.text = `Tailing Session: ${timeString}, ${this._eventRate} events/sec, Sampled: ${sampledString}`
```

Comment:
would it make sense to show "Sampled" only if it's being sampled, instead of Yes/No?

also can save space by avoiding unnecessary words like "session"

```suggestion
        this.statusBarItem.text = `Tailing: ${timeString}, ${this._eventRate} events/sec, Sampled: ${sampledString}`
```

Reply:
Our current CLI Experience always shows "Sampled: Yes/No" so to keep parity with that we will want to keep it displayed. I can remove Session though!

Reply:
It's good to avoid needless entropy, but when it comes to the IDE statusbar, matching the cli is very low on the priorities. Using space economically is very important here.

Reply:
Makes sense. Though Toolkit UX did review this as is.

Another point to consider: if a tailing session is fluctuating between being sampled or not (which can happen depending on the ingestion rate of log events into the LogGroup), a user would see the length/content of the statusBar change quite a bit if the word "Sampled" is appearing and disappearing. 

Changing between Yes and No would be a lot less jarring. Also seeing "Sampled: No" still tells me something about my session, whereas showing nothing at all doesn't tell me anything.